src/components/Mission/MissionDetails.js

```diff
@@ -0,0 +1,65 @@
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import FetchMission from '../../store/missions/missionsAPI';
+import { GetMissions, JoinMissions, LeaveMissions } from '../../store/missions/missionsReducer';
+
+const MissionDetails = () => {
+  const MissionStore = useSelector((store) => store.missions);
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    if (MissionStore.length === 0) {
+      FetchMission()
+        .then((response) => dispatch(GetMissions(response)));
+    }
```

Comment:
- [ ] The dispatch is not correctly configured, I can reserve the missions and rockets but when, I refresh the page is no longer there.